data/cube/Query.js

```diff
@@ -89,17 +89,8 @@ export class Query {
     // Implementation
     //---------------------------
     parseFields(names) {
-        const cubeFields = this.cube.fields;
```

Comment:
See the benefit of updating to ES6/lodash, just want to confirm that this safety check is unnecessary/unwarranted

Reply:
This was probably me going overboard trying to clean things up.  Agree the safety check is probably worth it.  Will restore post-merge